### PR TITLE
core: layout ie/performance fixes

### PR DESCRIPTION
**Pull request by coryrylan**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the new behavior?
- new cdn core demo
- new snowpack core demo
- new partial ie11 core demo (css only)
- bugfixes for safari/layouts with forms
- layout performance improvements
- better ie11/safari layout support
- responsive alignments

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: apps/core-ie/index.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
```
Comment (coryrylan) [inline apps/core-ie/index.html:1]: This demo is partially done. It demonstrates the global CSS and CSS custom properties work in IE. A followup PR will have demo of the web components.

File: packages/core/src/styles/layout/_type-grid.scss [hunk 2]
```diff
@@ -2,24 +2,16 @@
...
-  }
-}
-
 @mixin generateColumnsAndRows($breakpoint: null) {
   @for $i from 1 through $cds-token-layout-grid-cols-static {
     // implicit columns/rows
-    [cds-layout~='cols#{$breakpoint}:#{$i}'] > *,
-    [cds-layout~='cols#{$breakpoint}:#{$i}']::slotted(*) {
+    [#{$layout}*='#{$cols}#{$breakpoint}:#{$i}'] > *,
+    [#{$layout}*='#{$cols}#{$breakpoint}:#{$i}']::slotted(*) {
```
Comment (coryrylan) [inline packages/core/src/styles/layout/_type-grid.scss:9]: Selector names are in one file now which makes it easier to test optimizations and allow overrides. Example some systems may have strict `data-*` prefix requirements. The `*` attr selector also has a noticeable perf improvements on bundle compression size and runtime rendering.

File: packages/core/src/styles/module.shims.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+@import './layout/shims/grid';
+@import './layout/shims/gap';
+
+// older safari
+@media not all and (min-resolution: 0.001dpcm) {
+  @include generateGapShim();
+}
+
+// ie11
+@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
```
Comment (coryrylan) [inline packages/core/src/styles/module.shims.scss:19]: todo: these could be refactored into browser mixins

File: apps/core-ie/package.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  "version": "0.0.0",
+  "description": "",
+  "license": "MIT",
+  "scripts": {
+    "start": "lite-server"
+  },
+  "dependencies": {
+    "@clr/city": "^1.1.0",
+    "@clr/core": "./packages/core/dist/core",
+    "normalize.css": "^8.0.1"
```
Comment (jeeyun) [inline apps/core-ie/package.json:12]: nit: seems like we are using cdn for including `normalize.css` in this app so this dependency is not needed?

File: apps/core-snowpack/package.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "core-snowpack",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "snowpack dev",
+    "build": "snowpack build"
+  },
+  "dependencies": {
+    "normalize.css": "^8.0.1",
```
Comment (jeeyun) [inline apps/core-snowpack/package.json:10]: same question as above about npm dependency vs consuming from cdn.

File: apps/core-ie/package.json (hunk 4 above)
Comment (coryrylan) [reply apps/core-ie/package.json:12]: Good catch! I'll get this in my follow up bugfix pr

File: apps/core-snowpack/package.json (hunk 5 above)
Comment (coryrylan) [reply apps/core-snowpack/package.json:10]: This demo using the cdn for the css should be temporary. Waiting for a bugfix on snowpack where it wasnt importing css from node_modules properly.

**Pull request merged**